framework/elements/tree/function_to_tree.py

```diff
@@ -0,0 +1,195 @@
+
+
+
+class StringToTree:
+
```

Comment:
Camel case ( i.e paren_marker) variables and function names

Reply:
That's snake case. Do I still change to snake. Everything is already in camel (i.e parenMarker)

Reply:
Sorry yes change to snake case. everything except classnames and constants